suse_migration_services/prechecks/sshd.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2025 SUSE LLC.  All rights reserved.
+#
+# This file is part of suse-migration-services.
+#
+# suse-migration-services is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# suse-migration-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with suse-migration-services. If not, see <http://www.gnu.org/licenses/>#
+import logging
+import re
+
+from suse_migration_services.defaults import Defaults
+from suse_migration_services.command import Command
+
+log = logging.getLogger(Defaults.get_migration_log_name())
+
+
+def root_login(migration_system=False):
+    """
+    Check whether sshd is enabled and root login is allowed. If so,
+    warn user that they might want to enable it back before starting
+    the migration.
+    """
+    system_root = Defaults.get_system_root_path() if migration_system else '/'
```

Comment:
This code runs as pre-check on the host as well as during migration. During migration the DMS sets up the ssh capabilities itself and it is immaterial to check for them as they are all gone after migration. So correct me when wrong but the pre-check only makes sense on the host prior migration and then never again ?

If so, please differentiate this case and add

```python3
if migration_system:
    # This check should not run during migration
    return
```

Reply:
Fixed. I was thinking about this as well, thanks for pointing out!